frontend/src/app/update-file/update-file.component.ts

```diff
@@ -51,6 +55,26 @@ export class UpdateFileComponent implements OnChanges {
       this.selectedFiles = [currentValue];
     }
   }
+  readURL(event:any) {
```

Comment:
comment: avoid `any` in  type declarations.

See:
https://google.github.io/styleguide/tsguide.html#any

Reply:
Please fix.